arch/arm/core/aarch32/cortex_m/Kconfig

```diff
@@ -339,6 +339,13 @@ config CORTEX_M_DWT
 config TEST_EXTRA_STACK_SIZE
 	default 512 if TEST_ARM_CORTEX_M && FPU_SHARING
 
+config TRAP_UNALIGNED_ACCESS
+	bool "Enable unaligned access trap"
```

Comment:
Please no "Enable" on boolean Kconfig options, see https://github.com/zephyrproject-rtos/zephyr/pull/43586/

Reply:
Oky, renamed.

Reply:
might be better to do a batch fix later since this file is full of "enable"s.

https://github.com/zephyrproject-rtos/zephyr/blob/d510663c26e81f0490b4fcb4d39c102f95a408ba/arch/arm/core/aarch32/cortex_m/Kconfig#L282-L283

https://github.com/zephyrproject-rtos/zephyr/blob/d510663c26e81f0490b4fcb4d39c102f95a408ba/arch/arm/core/aarch32/cortex_m/Kconfig#L298-L299

https://github.com/zephyrproject-rtos/zephyr/blob/d510663c26e81f0490b4fcb4d39c102f95a408ba/arch/arm/core/aarch32/cortex_m/Kconfig#L309-L310

and more